integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/TransferDashFragment.kt

```diff
@@ -217,14 +217,17 @@ class TransferDashFragment : Fragment(R.layout.transfer_dash_fragment) {
                     binding.transferView.walletToCoinbase
 
             lifecycleScope.launch {
-                val details = transactionDetails.estimateNetworkFee(transferDashViewModel.dashAddress, dashValue,emptyWallet = isEmptyWallet)
+                val details = transferDashViewModel.estimateNetworkFee(dashValue, emptyWallet = isEmptyWallet)
                 val amountStr = details.amountToSend.toPlainString()
 
                 val isTransactionConfirmed = confirmTransactionLauncher.showTransactionDetailsPreview(
                     requireActivity(), address, amountStr, amountFiat, fiatSymbol, details.fee,
                     details.totalAmount, null, null, null)
-                if (isTransactionConfirmed){
-                    transferDashViewModel.sendDash(dashValue,isEmptyWallet)
+
+                if (isTransactionConfirmed) {
+                    AdaptiveDialog.withProgress(getString(R.string.please_wait_title), requireActivity()) {
```

Comment:
Wrapping a suspend function in `withProgress` helps to avoid a lot of issues with stuck progress dialogs.